src/MediaWiki/Hooks.php

```diff
@@ -993,11 +993,17 @@ public function onSkinTemplateNavigation( &$skinTemplate, &$links ) {
 	 */
 	public function onLoadExtensionSchemaUpdates( $databaseUpdater ) {
 
+		$applicationFactory = ApplicationFactory::getInstance();
```

Comment:
A bit weird you introduce this variable. Not needed. Certainly not this early in the function.